Extension/src/LanguageServer/configurations.ts

```diff
@@ -1744,7 +1748,10 @@ export class CppProperties {
             }
 
             if (!pathExists) {
-                const message: string = localize('cannot.find', "Cannot find: {0}", resolvedPath);
+                let message: string = localize('cannot.find', "Cannot find: {0}", resolvedPath);
+                if (quotedPath) {
+                    message += '. ' + localize('wrapped.with.quotes', 'Do not add extra quotes around paths.');
```

Comment:
Can you move the "." into the previous string? `"Cannot find: {0}."`? That prevents any issues in which the translator could add some additional punctuation that conflicts with the "." -- i.e. I've seen them add/remove periods.

Reply:
Same with the case below.

Reply:
I didn't want to add a period in the normal case. It appears to have been left out intentionally, perhaps to avoid confusion being attached to a path. Do you feel strongly about this feedback?